pkg/config/collector/options.go

```diff
@@ -35,17 +47,17 @@ func WithBootCMDLineFile(s string) Option {
 		return nil
 	}
 }
-func Directories(d ...string) Option {
+
+func StrictValidation(v bool) Option {
```

Comment:
(Probably on another PR) We should find a way for the collector options to only hold values that make sense in the context of the collector. Since the collector is not doing any validation (other than yamls being valid etc), this option doesn't belong here.

I think the problem is that the consumers (e.g. kairos, kcrypt etc), hijack the collector options to carry their own options like this one. Let's try to fix this on the next iteration.

Reply:
Yes agree, will open a generic issue and put this one as an example

Reply:
https://github.com/kairos-io/kairos/issues/1187